src/test/java/org/found_automation/SearchPage.java

```diff
@@ -0,0 +1,56 @@
+package org.found_automation;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import static org.found_automation.SearchTest.driver;
+
+
+public class SearchPage {
+
+    public WebDriver driver;
+
+    String inputValue = "automation";
```

Comment:
Use 'private' in this case. Because you use var only here.
Also use 'final' var